Doc/library/functions.rst

```diff
@@ -892,6 +892,10 @@ are always available.  They are listed here in alphabetical order.
    sequence (such as a string, bytes, tuple, list, or range) or a collection
    (such as a dictionary, set, or frozen set).
 
+   **CPython implementation detail:**  ``len`` raises :exc:`OverflowError`
+   on lengths larger than :data:`sys.maxsize`, such as
+   :class:`range(2 ** 64) <range>`.
```

Comment:
I suggest to use a number even larger, since some developers may expect maxsize to be equal to 2**64.

```suggestion
   :class:`range(2 ** 100) <range>`.
```

With `2**100`, there is a lower risk of confusion ;-)